tests/Resources/Page/ODH/JupyterHub/AccessGroups.resource

```diff
@@ -0,0 +1,161 @@
+*** Settings ***
+Resource      ../../../Page/Components/Components.resource
+Resource      ../../../Page/OCPDashboard/UserManagement/Groups.robot
+Resource      ../../../Common.robot
+Resource      JupyterHubSpawner.robot
+Resource      HighAvailability.robot
+Library       JupyterLibrary
+Library       OpenShiftCLI
+
+
+*** Variables ***
+${CUSTOM_ADMINS_GROUP}=   custom-admins-group
+${CUSTOM_USERS_GROUP}=    custom-users-group
+${STANDARD_ADMINS_GROUP}=  rhods-admins
```

Comment:
I saw that ${STANDARD_ADMINS_GROUP} and ${STANDARD_USERS_GROUP} are set in `Set Standard RHODS Groups Variables` depending on the RHODS version. Maybe you could add a note about this. Also, you could set here the new values (dedicated-admins, ...)

Reply:
yeah okay I can set here the new values

Reply:
done